schemas/mozilla_nimbus_schemas/experimenter_apis/experiments_v7/experiments_v7.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+
+from pydantic import BaseModel, ConfigDict, Field
+from pydantic.json_schema import SkipJsonSchema
+
+from mozilla_nimbus_schemas.experimenter_apis.common import (
+    ExperimentBucketConfig,
+    ExperimentLocalizations,
+    ExperimentOutcome,
+    _CommonBaseExperimentBranch,
+)
+
+
+class BaseExperimentV7(BaseModel):
+    """The base experiment definition for V7."""
+
+    schemaVersion: str = Field(description="Version of the NimbusExperiment schema.")
```

Comment:
We previously discussed moving all the comomn fields between the v6 and v7 api into a separate model inheriting from that. What happened?